worker/workercmd/worker.go

```diff
@@ -99,19 +98,17 @@ func (cmd *WorkerCommand) Runner(args []string) (ifrit.Runner, error) {
 
 	beaconRunner := worker.NewBeaconRunner(
 		logger.Session("beacon-runner"),
-		tsaClient,
+		&tsaClient,
```

Comment:
what happend here?

Reply:
looks like a sneaky gremlin cause I was working on 2 machines :P
its not in the final set of changes.